flow/envs/base.py

```diff
@@ -162,6 +162,9 @@ def __init__(self,
         # initial the vehicles kernel using the VehicleParams object
         self.k.vehicle.initialize(deepcopy(self.network.vehicles))
 
+        # if self.k.lane_area_detector:
+        #     self.k.lane_area_detector.initialize(deepcopy(self.network.template_detectors))
```

Comment:
What should be here?

Reply:
Hi Eugene,
I will try to take care of your requests on the weekend. I am happy to be able to contribute.